wgpu-hal/src/lib.rs

```diff
@@ -1757,7 +1757,10 @@ bitflags!(
     #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
```

Comment:
above comment `//TODO: it's not intuitive for the backends to consider LOAD being optional.` is now outdated